simul/runfiles/test_randhound.toml

```diff
@@ -0,0 +1,9 @@
+Servers = 10
+Simulation = "RandHound"
+BF = 2
+Rounds = 1
+Trustees = 5
+Purpose = "RandHound Test"
+
+Hosts
+10
```

Comment:
No, that's a copy&paste fragment. It could make sense to be able to have a list of host-trustee configurations though.